### PR TITLE
Finetune SSD VGG

### PR DESCRIPTION
**Pull request by borisneal**
Wrapper code and notebooks to finetune SSD VGG based object detector and deploy it on FPGA instances.

### CODE REVIEW

File: how-to-use-azureml/deployment/accelerated-models/finetune-ssd-vgg/notebooks/Finetune VGG SSD.ipynb [hunk 1]
```diff
@@ -0,0 +1,423 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Finetuning VGG-SSD Object Detection Model"
```
Comment (paledger) [inline how-to-use-azureml/deployment/accelerated-models/finetune-ssd-vgg/notebooks/Finetune VGG SSD.ipynb]: Could you add a pointer to this notebook from the [SSD-VGG Quickstart notebook](https://github.com/borisneal/MachineLearningNotebooks/blob/master/how-to-use-azureml/deployment/accelerated-models/accelerated-models-object-detection.ipynb)?

File: how-to-use-azureml/deployment/accelerated-models/finetune-ssd-vgg/notebooks/Deploy Accelerated.ipynb [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azureml.core import Workspace\n",
+    "\n",
+    "ws = Workspace.from_config(path=\"./config.json\")\n",
```
[the remark this answers is not in the document]
Comment (Zach-Neal-Analytics) [reply how-to-use-azureml/deployment/accelerated-models/finetune-ssd-vgg/notebooks/Deploy Accelerated.ipynb]: OK.  Will be updated in latest changes.

File: how-to-use-azureml/deployment/accelerated-models/finetune-ssd-vgg/notebooks/Deploy Accelerated.ipynb [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azureml.core.model import Model\n",
+    "from azureml.core.image import Image\n",
+    "from azureml.accel import AccelOnnxConverter\n",
+    "from azureml.accel import AccelContainerImage\n",
+    "\n",
+    "model_ckpt_dir = r'/datadrive/Dropbox/neal/kroger/azml_ssd_vgg/'\n",
```
[the remark this answers is not in the document]
Comment (Zach-Neal-Analytics) [reply how-to-use-azureml/deployment/accelerated-models/finetune-ssd-vgg/notebooks/Deploy Accelerated.ipynb]: OK.  Now I have the user create the folder in the Fintetune VGG SSD notebook.  The user then references the same folder here.

File: how-to-use-azureml/deployment/accelerated-models/finetune-ssd-vgg/notebooks/Deploy Accelerated.ipynb [hunk 4]
```diff
@@ -0,0 +1,302 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import glob\n",
+    "\n",
+    "image_dir = r'/datadrive/Dropbox/neal/data/kroger/dataset/test/JPEGImages/'\n",
```
[the remark this answers is not in the document]
Comment (Zach-Neal-Analytics) [reply how-to-use-azureml/deployment/accelerated-models/finetune-ssd-vgg/notebooks/Deploy Accelerated.ipynb]: OK.  Changed the reference to a generic folder that is created in the Finetune VGG SSD notebook image_dir = os.path.join(home, 'azml_ssd_vgg', 'JPEGImages')

**Pull request merged**